src/ui_basic/listselect.cc

```diff
@@ -58,6 +60,14 @@ BaseListselect::EntryRecord::EntryRecord(const std::string& init_name,
 	   UI::g_fh->render(as_richtext_paragraph(richtext_escape(hotkey_text), style.hotkey()));
 }
 
+inline BaseListselect::EntryRecord::~EntryRecord() {
```

Comment:
Sorry but inline makes only sense in a .h file I wonder that no compiler complains here :-)

Reply:
`EntryRecord` is only really used within this one source file so it actually can be inlined. Using it in other .cc files as well might cause linker errors